lib/logsource.c

```diff
@@ -457,6 +457,57 @@ _create_ack_tracker_if_not_exists(LogSource *self)
     }
 }
 
+static void
+_register_raw_bytes_stats(LogSource *self)
+{
```

Comment:
As we talked about it IRL, this should be optional, so we do not create unused counters for Fetchers, etc.